content/docs/building-apps/first-deposit.mdx

```diff
@@ -0,0 +1,58 @@
+
+The flow with Claimable Balances looks like this, 
+
+1. The wallet registers a new user, and generates a keypair 
+2. The wallet initiates a deposit on behalf of a user
+3. The anchor provides deposit instructions to the wallet 
+4. The user transfers money from a bank account to the anchor’s account
+5. The anchor creates and funds the user's Stellar account plus the amount required for trustlines and transaction fees.  Again, we suggest 2 XLM to start. 
+6. The anchor creates a Claimable Balance 
+7. The wallet detects the Claimable Balance for the account, claims the funds, and posts it in the wallet. 
+
+![anchor creates account claimable balance flow](../../web/assets/first-deposit-claimable-balance-flow.png)
+
+### Option two: The wallet creates and funds the Stellar account upon user sign-up. 
+
+For this option, the wallet creates and funds the Stellar account upon every new user sign-up with the minimum requirement of 1XLM, plus the .5XLM reserve for establishing the first trustline, plus a bit more to cover transaction fees. For more information on minimum balance, check out this [page](https://developers.stellar.org/docs/glossary/minimum-balance/). 
+
+The flow looks like this: 
+
+1. Upon a new user signup, the wallet issues a keypair, then creates and funds the user's Stellar account with 2XLM. 
+2. Then the wallet creates a trustline, and initiates the first deposit 
+3. Once the deposit request is sent to the anchor, the anchor provides instructions for the deposit. 
+4. The customer transfer funds from a personal bank account to the anchor’s account
+5. The anchor receives the funds, then sends them to the user’s Stellar account 
+6. The wallet detects that funds were sent and notifies the user. 
+
+![wallet creates account flow](../../web/assets/first-deposit-wallet-flow.png)
+
+*Note*: in the examples above, we suggest having the anchor or wallet cover minimum balance and trustline XLM requirements by depositing funds directly into a user's account.  We made that suggestion for the sake of simplicity, but in all cases, the anchor or wallet could instead use Sponsored Reserves to ensure that when a user closes a trustline or merges their account, the reserve reverts to the sponsoring account rather than to the user's account.  For more information on how that works, see the [Sponsored Reserves](https://developers.stellar.org/docs/glossary/sponsored-reserves/) documentation. 
```

Comment:
```suggestion
**Note**: In the examples above, we suggest having the anchor or wallet cover minimum balance and trustline XLM requirements by depositing funds directly into a user's account.  We made that suggestion for the sake of simplicity, but in all cases, the anchor or wallet could instead use [Sponsored Reserves](../glossary/sponsored-reserves.mdx) to ensure that when a user closes a trustline or merges their account, the reserve reverts to the sponsoring account rather than to the user's account.
```
Since we're already referencing Sponsored Reserves directly, we don't need an explicit "For more" sentence imo and can inline the cross-reference.